volume/local/local.go

```diff
@@ -90,6 +96,7 @@ func (r *Root) Create(name string) (volume.Volume, error) {
 	return v, nil
 }
 
+// Remove the directory with all contents
```

Comment:
Remove removes the specified volume and all underlying data. If the given volume does not belong to this driver and error is returned. The volume is reference counted, if all all references are not released then the volume is not removed.